src/types.ts

```diff
@@ -28,14 +28,85 @@ export interface NostoClient {
  */
 export interface Recommendation {
   result_id: string
-  products: Product[]
+  products: PushedProduct[]
```

Comment:
Product is too minimal for this use case
PushedProduct matches the signature better